### ISSUE TITLE
Missed whitespace

### ISSUE THREAD

**Issue by ai** [postcss/postcss-custom-properties#137]
Input CSS:

```css
  .base {
    box-shadow: inset 0 -3px 0 var(--buttonDepthColor);
  }
```

Output CSS:

```css
  .base {
     box-shadow: inset 0 -3px 0rgba(0, 0, 0, 0.2);
  }
```

/cc @jonathantneal

**Comment by jonathantneal** [comment]
Thanks for reporting this. It is resolved in v8.0.2.

**Comment by ai** [comment]
Thanks, it was fast :)
